### DISCUSSION TITLE
How to get a list of all stories in either array or json form

### DISCUSSION THREAD

**Discussion by PaulACoroneos** [storybookjs/storybook#28720] [Help]
### Summary

Hey all. I am trying to find a way to generate a list (or object) of all storybook stories that are in my repo. I want to take this list then write automation to validate each story at least renders to prevent regressions. I saw some stuff about sb extract but I didn't fully grok how to use it. Thank you so much.

**Comment by smlimasu** [comment]
https://github.com/storybookjs/storybook/discussions/28195 maybe this helps
